docs/guide/New-Subscriber-via-webconsole.md

```diff
@@ -1,38 +1,42 @@
 <!-- Google tag (gtag.js) --> <script async src="https://www.googletagmanager.com/gtag/js?id=G-JETJ7TJ805"></script> <script> window.dataLayer = window.dataLayer || []; function gtag(){dataLayer.push(arguments);} gtag('js', new Date()); gtag('config', 'G-JETJ7TJ805'); </script>
 
-## New Subscriber via webconsole
+# New Subscriber via Webconsole
 
-### 1. [Install webconsole](https://github.com/free5gc/free5gc/wiki/Installation#d-install-webconsole)
+## 1. Install Webconsole
 
-### 2. (Optional)Delete MongoDB 
+If Webconsole isn't installed yet, please, follow the instructions from [this page](./3-install-free5gc.md#d-install-webconsole).
+
+## 2. (Optional) Delete MongoDB database
 
 If another version of free5GC was ran before, you have to delete MongoDB.
+```
+$ mongo --eval "db.dropDatabase()" free5gc
+```
     
-    $ mongo --eval "db.dropDatabase()" free5gc
-    
-### 3. Run WebConsole server
+## 3. Run Webconsole server
 ```
 $ cd ~/free5gc/webconsole
-$ ./bin/webconsole
+$ go run server.go
```

Comment:
Should rollback it. After installing webconsole, there will be ./bin/webconsole binary.

Reply:
I changed it because of the command displayed in [this section](https://free5gc.org/guide/5-install-ueransim/#4-use-webconsole-to-add-an-ue) of the docs. Probably we should revert my change here and add another one there, right?

Reply:
ok, we'll update it later.